src/prototypes/article-listing/example/example.twig

```diff
@@ -0,0 +1,145 @@
+<div id="article-listing">
+  <div class="o-container o-container--pad">
+    <div class="o-container__content">
+      {% embed '@cloudfour/objects/deck/deck.twig' with {
+        class: 'o-deck--3-column@l'
+      } %}
+        {% block content %}
+          {% for article in recent.articles %}
+            {% if loop.index == '1' %}
+              {% set _card_class = 'c-card--horizontal@l' %}
+            {% else %}
+              {% set _card_class = '' %}
+            {% endif %}
+            {% embed '@cloudfour/components/card/card.twig' with {
+              class: _card_class,
+              href: article.link
+            } %}
+              {% block heading %}
+                {{article.title}}
+              {% endblock %}
+              {% block cover %}
+                <img src="{{article.thumbnail}}" alt="">
+              {% endblock %}
+              {% block content %}
+                <p>
+                  {% if article.description|length > 140 %}
+                    {{article.description|slice(0, 140)|trim}}…
+                  {% else %}
+                    {{article.description}}
+                  {% endif %}
+                </p>
+              {% endblock %}
+              {% block footer %}
+                <div class="c-author o-media">
+                  <div class="o-media__object">
+                    {% include '@cloudfour/components/avatar/avatar.twig' with { src: article.avatar } only %}
+                  </div>
+                  <div class="o-media__content">
+                    <p<a href="#">{{article.author}}</a></p>
+                    <p>{{article.pubDate|date('M j, Y')}}</p>
+                  </div>
+                </div>
+              {% endblock %}
+            {% endembed %}
+          {% endfor %}
+        {% endblock %}
+      {% endembed %}
+    </div>
+  </div>
+  {% for topic in featured.topics %}
+    <div class="o-container o-container--pad">
+      <div class="o-container__content">
+        {% embed '@cloudfour/objects/deck/deck.twig' with {
+          class: 'o-deck--3-column@l',
+        } %}
+          {% block content %}
+            <div class="title-content">
+              <h2 class="section-title">{{topic.title}}</h2>
+              <div>
```

Comment:
Can these topic buttons move to _after_ the article listings on small screens rather than after the headings?